### DISCUSSION TITLE
Specify nested property using other property values

### DISCUSSION THREAD

**Discussion by PurplePixieX** [mProjectsCode/obsidian-meta-bind-plugin#312] [Q&A]
Looking to be able to use VIEW to dynamically access different data depending on other properties. How can I do this?

```
---
nest:
    isNested: egg
key: isNested
---
```
`VIEW[{nest[key]}]` gives an error.

**Comment by mProjectsCode** [comment]
Only with JavaScript, see the advanced examples in the docs.
